qiskit/ignis/verification/randomized_benchmarking/circuits.py

```diff
@@ -351,6 +354,9 @@ def randomized_benchmarking_seq(nseeds: int = 1,
 
             interleaved_elem = [qc_cx, qc_x, qc_h]
 
+            # or as gate objects:
+            interleaved_elem = [CXGate(0,1), XGate(0), HGate(0)]
```

Comment:
When you use gate objects you cant specify which qubits they act on so i think this should be
```suggestion
            interleaved_elem = [CXGate(), XGate(), HGate()]
```